config/config.go

```diff
@@ -120,12 +121,6 @@ func InitConfig() *viper.Viper {
 		//panic(fmt.Errorf("Fatal error config file: %s \n", err))
 	}
 
-	//SetFlags(v)
-
```

Comment:
This this can be un-removed. You need a local.yaml in examples/*/local.yaml and then run `go run cmd/*/main.go`